### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by alvaroiporre**
**Project General Changes**

- [x] Added app.js file.
- [x] Added javascript linters.
- [x] Added the nav menu for mobile.
- [x] Modified classes and IDs to manipulate DOM from JS .

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -33,23 +33,64 @@ body {
...
+  backdrop-filter: blur(5px);
+  width: 100%;
+  height: 100%;
+  text-align: left;
+  transition: 0.3s;
+}
+
+.nav_li_item {
+  text-decoration: none;
+  font-weight: 500;
```
Comment (huseyinbiyik) [inline styles.css:86]: Great job with the font size 👍  But let's stick to the Figma design guideline and fix the font weight. It needs to be 600.
Screenshot 2023-04-10 at 11 43 03 PM
Screenshot 2023-04-10 at 11 44 32 PM

File: styles.css [hunk 2]
```diff
@@ -33,23 +33,64 @@ body {
...
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(5px);
+  width: 100%;
+  height: 100%;
+  text-align: left;
+  transition: 0.3s;
+}
+
+.nav_li_item {
```
Comment (huseyinbiyik) [inline styles.css:84]: Your nav item is too close to each other. Kindly give some space between the nav items like on the Figma template. 👍 
Screenshot 2023-04-10 at 11 42 06 PM
Screenshot 2023-04-10 at 11 42 51 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (alvaroiporre) [reply styles.css:85]: thank you @huseyinbiyik , fixed changes.

**Pull request merged**